luarules/gadgets/game_custom_map_tidal.lua

```diff
@@ -0,0 +1,25 @@
+function gadget:GetInfo()
+	return {
+		name = "Custom Map Tidal",
+		desc = "Sets map tidal for modoption via engine call",
+		author = "",
+		date = "",
+		license = "Horses",
+		layer = 0,
+		enabled = true  --  loaded by default?
+	}
+end
+
+if gadgetHandler:IsSyncedCode() then
```

Comment:
right now the unsynced portion includes the `GetInfo` call but it's useless. put this at the top (ideally in the form of `if not then return false`)